protocol/xmrtaker/message_handler.go

```diff
@@ -296,32 +296,10 @@ func (s *swapState) handleNotifyXMRLock(msg *message.NotifyXMRLock) (net.Message
 		select {
 		case <-s.ctx.Done():
 			return
+		// TODO: document why we add one second
```

Comment:
@noot What comment can I add here to explain why we add a second?

Reply:
a second is added here so that we don't refund at the *exact* moment we are able to (ie. exactly at t1) as the block time of the network might vary and potentially be slightly behind, so our tx might get rejected